plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreAuthenticationModule.java

```diff
@@ -36,6 +37,8 @@ protected void setup(Binder binder)
     {
         newOptionalBinder(binder, IdentityAwareMetastoreClientFactory.class)
                 .setDefault().to(UgiBasedMetastoreClientFactory.class).in(SINGLETON);
+        newExporter(binder).export(IdentityAwareMetastoreClientFactory.class)
```

Comment:
You are missing in `io.trino.tests.product.deltalake.TestDeltaLakeJmx#testJmxTablesExposedByDeltaLakeConnectorBackedByThriftMetastore` the new jmx table
`io.trino.plugin.hive.metastore.thrift:name=delta,type=thriftmetastorestats`